Logger/FileLogger.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Text;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        public String filePath { get; }
```

Comment:
Properties should be PascalCase.
You want to favor using C# keywords rather than class names.
```suggestion
        public string FilePath { get; }
```